scripts/src/java/org/oppia/android/scripts/proto/script_exemptions.proto

```diff
@@ -5,18 +5,17 @@ package proto;
 option java_package = "org.oppia.android.scripts.proto";
 option java_multiple_files = true;
 
-// Exemptions for the script checks.
-// An exemption is an entity that should not be tested for a corresponding script check.
-// In general the exemptions are files, which should be skipped for any particular
-// script check. Also, note that the exemptions are maintained lexicographically sorted in the
-// textproto files. To add any new exemption, please add it at the correct lexicographical
-// position, so that the list remains sorted.
-message ScriptExemptions {
-  // List of all the exemptions.
-  // For exempting any file for a script check, provide its relative path to root in the
-  // corresponding textproto file of the script exemption.
-  // For example if we want 'ActivityComponent.kt' file to be exempted for the test file check, then
-  // add 'exemption: "app/src/main/java/org/oppia/android/app/activity/ActivityComponent.kt"' at
-  // the correct lexicographical position of the 'test_file_exemptions.textproto' file.
-  repeated string exemption = 1;
+// Exemptions for the test-file check.
+// Here, exemption implies those files, which we do not wish to check for the test-file presence.
```

Comment:
Nit: combine these into one doc comment rather than having the extra line between them.

Though, please do wrap at the 100 character limit like normal.

Also, nit on phrasing:

```
Exemptions indicate files for which we should not check for having test files.
```

(Slightly reworded for clarity).

Reply:
Done.